static/_redirects

```diff
@@ -1,7 +1,9 @@
 #redirect file for moved web pages.
-/main-concepts                              /tooling
-/history                                    /research
-/cosign/overview                            /signing/quickstart    
+/main-concepts                              /about/tooling
+/security                              /about/security
+/threat-model                              /about/threat-model/
```

Comment:
```suggestion
/threat-model                               /about/threat-model/
```

Reply:
Looks good. Going to have to be multiple commits as GH isn't letting me batch (probably related to the signoff).